### PR TITLE
add srijan

### PR DESCRIPTION
**Pull request by srijan-vaddadi**

### CODE REVIEW

**Review by nmcc1212 — DISMISSED** [review]

**Comment by SSG27** [comment]
has unresolved conflicts
aamirahs just got approved so u gotta redo urs

**Review by SSG27 — DISMISSED** [review]

**Comment by SSG27** [comment]
merge conflicts

**Review by AamirahP — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -20,7 +20,9 @@
   "keywords": [],
   "contributors": [
     { "name": "Fergus Bentley", "email": "mail@fergcb.uk", "url": "https://fergcb.uk" },
-    { "name": "Aamirah Patel", "email": "aamirahpatel@hotmail.co.uk", "url": "https://github.com/AamirahP" }
+    { "name": "Nial", "email": "nial@n1al.com", "url": "https://n1al.com" },
```
Comment (nmcc1212) [inline package.json:23]: im here twice?

File: package.json [hunk 2]
```diff
@@ -20,7 +20,9 @@
...
+    { "name": "Srijan Vaddadi", "email": "srijan.vaddadi@gmail.com", "url": "https://srijan.uk" },
+    { "name": "Aamirah Patel", "email": "aamirahpatel@hotmail.co.uk", "url": "https://github.com/AamirahP" },
     { "name": "Joshua Mushaikwa", "email": "jjnotm@gmail.com", "url": "https://github.com/incogzino" },
     { "name": "Nial", "email": "nial@n1al.com", "url": "https://n1al.com" }
```
Comment (nmcc1212) [inline package.json:27]: 2nd time

**Review by nmcc1212 — CHANGES_REQUESTED** [review]

**Review by AamirahP — APPROVED** [review]

**Review by nmcc1212 — DISMISSED** [review]

**Review by AamirahP — APPROVED** [review]

**Review by SSG27 — CHANGES_REQUESTED** [review]
add a comma after your curly bracket

**Review by SSG27 — APPROVED** [review]

**Review by AamirahP — APPROVED** [review]

**Pull request merged**